### ISSUE TITLE
ETQ Instructeur, je veux pouvoir sauvegarder mes filtres

### ISSUE THREAD

**Issue by kara22** [demarches-simplifiees/demarches-simplifiees.fr#9464]

**Comment by LeSim** [comment]
pourquoi pas, mais pourquoi on priorise cette feature par rapport 
- a la roadmap ?
- a d'autres moyens de retrouver ces dossiers : tags / libellés / marquer comme non lu

J'ai l'impression qu'on va un peu vite la dessus (cf le nombre de mot d'un champ texte).

**Comment by Olivier-Marcellin** [comment]
oui, il faudrait cadrer mieux et lister les cas d'usages et  aussi interview des instructeurs

**Comment by kara22** [comment]
@Olivier-Marcellin qui s'en occupe pour les interviews instructeurs, on voit avec le support ?

**Comment by Olivier-Marcellin** [comment]
il faudrait des contact d'instructeurs via le support oui

**Comment by tchak** [comment]
Je ne comprends pas ce que ça veut dire. Les filtres sont déjà sauvegardés. On parle du fait d'ajouter de nouveaux onglets (avec le filtre personnalisable) ?
